client.go

```diff
@@ -120,7 +120,7 @@ func verifyFile(c *cli.Context) error {
 	if len(c.Args().First()) == 0 {
 		printErrAndExit("Please give the 'msgFile'", 1)
 	}
-	dbg.SetDebugVisible(c.GlobalInt("debug"))
+	log.SetDebugVisible(c.GlobalInt("debug"))
```

Comment:
Please think about this issues before you ask me to merge `ui` and `dbg` in one package called `log`. Propose a better name, please.